deploy/docs/Best_Practices.md

```diff
@@ -580,3 +584,31 @@ kube-prometheus-stack:  # For values.yaml
   prometheus:
     prometheusSpec:
       scrapeInterval: '1m'
+```
+
+## Get logs not available on stdout
+
+When logs from a Pod are not available on stdout, [Tailing Sidecar Operator](https://github.com/SumoLogic/tailing-sidecar)
```

Comment:
```suggestion
When logs from a pod are not available on stdout, [Tailing Sidecar Operator](https://github.com/SumoLogic/tailing-sidecar)
```
?

Reply:
I don't know how it should be written, In k8s docs Pod starts with capital letter, in other places it is written either as "Pod" or "pod"

Reply:
```
➜  sumologic-kubernetes-collection git:(drosiek-otc) ✗ grep ' pod ' **/**.md | wc -l
55
➜  sumologic-kubernetes-collection git:(drosiek-otc) ✗ grep ' Pod ' **/**.md | wc -l
7
```

I have no idea neither, by statistically we want to use `pod` :sweat_smile: